src/src/com/microsoft/aad/adal/AuthenticationActivity.java

```diff
@@ -790,7 +785,7 @@ protected TokenTaskResult doInBackground(String... urlItems) {
                 // Record account in the AccountManager service
                 try {
                     setAccount(result);
-                } catch (Exception exc) {
+                } catch (GeneralSecurityException | IOException exc) {
```

Comment:
Sorry I'm not sure I understand. Do you prefer to write a message that clearly indicates a type of error in the syslog directly? I thought the reason why Logger was created is to make all tracing logic centralized (especially around exceptions and errors)... If we need to improve Logger we can do it in separate pull request. Can you please provide an example of desired code and some explanation on how it would help to debug an issue?